code/modules/hydroponics/growninedible.dm

```diff
@@ -13,10 +13,6 @@
 	/// The reagent this plant distill to. If NULL, it uses a generic fruit_wine reagent and adjusts its variables.
 	var/distill_reagent
 
-// This may look like it's doing nothing but it's necessary, we do this to have kwargs work in New (for passing into Initialize)  
-/obj/item/grown/New(loc, obj/item/seeds/new_seed)
```

Comment:
We did this for a reason see #78414. @vinylspiders thoughts on this still being nessassary?

Reply:
My understanding of the comment was that it was needed for named arguments specifically, which if true it'd be easy to just _not_ use those named arguments. But there being a pr specifically adding it back makes me question that.
I'll quickly go test the bugs mentioned in that PR, thanks for bringing it up.

Reply:
Well I wasn't able to find any immediate bugs with it. All the plants I harvested are interactable, logs are choppable, they all make seeds. There's a couple later overrides of initialize such as `/obj/item/food/grown/banana/Initialize(mapload)`, but the new_seed argument is still passed along despite not being referenced. I can't find any item/grown or food/grown initializers that use any other arguments which would conflict with new_seed

But, I'll draft this until I can get some feedback from someone more familiar with the problems that arose before. There were so many bugs last time that I'll just not bother if we're not 100% sure.

Reply:
Just scrapping the plant changes since I don't want to open that can of worms right now.